### PR TITLE
Prepare 1.12.5

### PR DESCRIPTION
**Pull request by gwarf**
Prepare 1.12.5 and fix some errors in the spec file.

Still to be done:
- [x] #2 should be merged before this PR.
- [x] update changelog and bump version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,17 +70,26 @@ git checkout X.X.X
 # Building in a container
 docker run --rm -v $(pwd):/source -it centos:7
 yum install -y rpm-build make rsync gcc
```
Comment (enolfc) [inline README.md]: If using builddep below then make rsync and gcc shouldn't be needed

**Review by enolfc — COMMENTED** [review]
Nice @gwarf! Just one minor comment

File: README.md (hunk 1 above)
Comment (enolfc) [reply README.md]: and probably yum-utils? is missing

Comment (gwarf) [reply README.md]: So builddep is indeed enough.
But it seems I can run it directly from within my docker, without having to manually install yum-utils that is still present.
Have you seen cases where it was not already available?

Comment (gwarf) [reply README.md]: I've addressed this, certainly safer to explicitly install yum-utils, thanks!

**Comment by gwarf** [comment]
So before merging this we need to figure why the manpage and the license file are not installed while they are in the RPM. (tested using `centos:7` and `centos:6` docker images)

File: README.md (hunk 1 above)
Comment (enolfc) [reply README.md]: just for the record, CentOS 7 has it, but 6 & 8 not:
```
$ docker run --rm -v -it centos:6 yum-builddep
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"yum-builddep\": executable file not found in $PATH": unknown.
$ docker run --rm -v -it centos:8 yum-builddep
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"yum-builddep\": executable file not found in $PATH": unknown.
```

So good that we explicitly add it

File: README.md [hunk 2]
```diff
@@ -49,25 +58,38 @@ The required build dependencies are:
...
 ```sh
 # Checkout tag to be packaged
 git clone https://github.com/EGI-Foundation/lcg-info.git
 cd lcg-info
 git checkout X.X.X
 # Building in a container
 docker run --rm -v $(pwd):/source -it centos:7
-yum install -y rpm-build make rsync gcc
+yum install -y rpm-build yum-utils
+yum-builddep -y lcg-info.spec
```
Comment (enolfc) [inline README.md:82]:
```suggestion
yum-builddep -y /source/lcg-info.spec
```

Comment (enolfc) [reply README.md:82]: that or a `cd /source` before

**Comment by gwarf** [comment]
> So before merging this we need to figure why the manpage and the license file are not installed while they are in the RPM. (tested using `centos:7` and `centos:6` docker images)

So it's due to https://github.com/CentOS/sig-cloud-instance-build/blob/98aa8c6f0290feeb94d86b52c561d70eabc7d942/docker/centos-6.ks#L80.
Installing with `rpm -i --includedocs lcg-info*.rpm` works fine.

File: README.md (hunk 1 above)
Comment (gwarf) [reply README.md]: Cool, thanks for the confirmation

**Review by enolfc — APPROVED** [review]
LGTM!

**Pull request merged**